spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ops/ModuleToOperation.scala

```diff
@@ -43,3 +43,32 @@ object ModuleToOperation {
     (implicit ev: TensorNumeric[T]): ModuleToOperation[T] =
     new ModuleToOperation(model.asInstanceOf[AbstractModule[Activity, Activity, T]])
 }
+
+class TensorModuleToTFModule[T: ClassTag, D: ClassTag] private
```

Comment:
rename it to TensorModuleToOperation? Also please add java document to explain why we need this.